ASM/README.md

```diff
@@ -1,11 +1,55 @@
-- Download the armips assembler: <https://github.com/Kingcom/armips>, build it if necessary, and put the executable in the `tools` directory, or somewhere in your PATH.
+Advanced modifications to the Randomzier source require a bit more software than what is needed for running it.
+
+## Assembly: armips
+### Windows Prerequisite
+- Download and install the [Visual Studio 2015-202x Visual C++ Redistributable](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022) package.
+  - You will want the x64 Architecture version.
+  - This is to run the automated build of armips. If you plan to compile it yourself you can ignore this, but that is an advanced setup not covered here.
+### Running
+- Download the armips assembler: <https://github.com/Kingcom/armips>
+  - [Windows automated builds](https://buildbot.orphis.net/armips/)
+  - On other platforms you'll need either `clang` or `gcc`, `cmake`, and either `ninja` or `make` installed. All of these should be available in the package repositories of every major Linux distribution and Homebrew on macOS. After, follow the [building from source instructions](https://github.com/Kingcom/armips#22-building-from-source).
+- Put the armips executable in the `tools` directory, or somewhere in your PATH.
 - Put the ROM you want to patch at `roms/base.z64`. This needs to be an uncompressed ROM; OoTRandomizer will produce one at ZOOTDEC.z64 when you run it with a compressed ROM.
 - Run `python build.py`, which will:
   - create `roms/patched.z64`
   - update some `txt` files in `build/` and in `../data/generated/`. Check `git status` to see which ones have changed. Make sure you submit them all together!
 
-To recompile the C modules, use the `--compile-c` option. This requires the N64 development tools to be installed: <https://github.com/glankk/n64>
+## C: n64 toolchain
+### Prerequisites
+Recompiling the C code for randomizer requires the N64 development tools to be installed: <https://github.com/glankk/n64>. There are several ways to do this depending on your platform.
+- **Windows**:
+  - **Without WSL**: [Download this zip archive](https://discord.com/channels/274180765816848384/442752384834469908/1085678948614144081) and extract the `n64` folder into the `tools` directory alongside armips.
+    - Download and install [MSYS2](https://www.msys2.org/#installation).
+      1. Accept the defaults in the installer.
```

Comment:
I would add an asterisk here that MSYS2 isn't really required. I just use base Windows cmd lol. But it is probably easier, so I don't mind leaving it as the default recommendation.

Reply:
It is required the way I have it written because the dropbox binaries are dynamically linked to the MSYS2 libraries and cannot work without them. If you make a new link with static linking MSYS2 won't be required, but would be larger and the libraries it depends on won't be updated if they decide to update their environments.

Reply:
Ah, fair enough.